playbooks/modules/nfs_server.yml

```
@@ -0,0 +1,63 @@
+---
+- name: NFS Server operations on Powerstore Storage System
+  hosts: localhost
+  connection: local
+  vars:
+    array_ip: "10.**.**.**"
+    validate_certs: false
+    user: "user"
+    password: "password"
+    nas_server_name: "sample_nas_1"
+
+  tasks:
+    - name: Enable NFS server
+      register: result
+      dellemc.powerstore.nfs_server:
+        array_ip: "{{ array_ip }}"
+        validate_certs: "{{ validate_certs }}"
+        user: "{{ user }}"
+        password: "{{ password }}"
+        nas_server: "{{ nas_server_name }}"
+        host_name: "sample_host_name"
```

Hostname usually will not have underscore.
```suggestion
        host_name: "sample.com"
```